src/src/Mastermind.java

```diff
@@ -1,16 +1,42 @@
+import java.util.ArrayList;
 
 class Mastermind {
+	private ArrayList<Attempt> attempts = new ArrayList<Attempt>();
+	private SecretCombination secretCombination;
+	private static final int NUM_ATTEMPS = 10;
+	
 
 	public Mastermind() {
-		
+		secretCombination = new SecretCombination();
 	}
 	
 	public void play() {
+		Console console = new Console();
+		int nAttemps = 0;
+		console.write(nAttemps + " attemp(s):");
+		console.write("xxxx");
```

Comment:
SecretCombination tiene un método toString(), lo que imprime es la xxxx
```suggestion
		console.write(this.secretCombination.toString());
```

Reply:
Y cómo saco el "0  attemps"? Lo pongo tal cual?

Reply:
Lo puedes sacar del size de Atenas, al inicio el size es 0